third_party/nvfuser/csrc/transform_iter.cpp

```diff
@@ -174,13 +174,8 @@ void ReplayTransformations::handle(Swizzle2D* swizzle_2d) {
 
 ReplayTransformations::ReplayTransformations(
     const std::vector<IterDomain*>& _target_domain,
```

Comment:
Unrelated question: why are their names starting with `_`? Is this some naming convention?

Reply:
I guess that's written by @csarofeen long time ago. Will change.